gitrise.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 # setting exit on error
```

Comment:
You don't need the `# setting exit on error` comment anymore.

Reply:
removed it